### PR TITLE
#162198142 Add continuous integration and code quality tools

### PR DESCRIPTION
**Pull request by andela-oawofadeju**
#### What does this PR do?
- This PR adds continuous integration and code quality tools to the Repo

#### Description of Task to be completed?
- There should be a continuous integration tool in place.
- There should be a continuous deployment pipeline.
- There should a code quality tool in place.
- We can use a combination Travis CI, CodeClimate and Coveralls.

#### How should this be manually tested?
- check the travis builds and codeclimate reports to see if the setup is done properly.

#### What are the relevant pivotal tracker stories?
- #162198142

### CODE REVIEW

**Comment by andela-oawofadeju** [comment]
Thanks @kevinsamoei. I can create a develop branch off of master. Also feedback noted for tests.

**Comment by james947** [comment]
@andela-oawofadeju I have created the develop branch you can do the change to have the base as develop

**Comment by james947** [comment]
@andela-oawofadeju Kindly remove the hyphens on the PR title. For more info [link](https://github.com/andela/engineering-playbook/tree/master/5.%20Developing/Conventions#pr-title)

**Comment by kevinsamoei** [comment]
@andela-oawofadeju One more thing, we advocate for one commit per PR and the conventions found in the engineering playbook be used for the commit message.

**Comment by andela-oawofadeju** [comment]
@kevin, I'm going to squash the commits when the review is done

File: .coveralls.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+repo_token: ebupaz14VF9KLKCRkqVKR42eVEJrx5ZzE
```
Comment (gr1d99) [inline .coveralls.yml]: can we have this token stored in an environment variable

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+before_install:
+  - export CHROME_BIN=/usr/bin/google-chrome
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+
+
+cache:
+  directories:
+    - "node_modules"
```
Comment (gr1d99) [inline .travis.yml]: Add a new line here

**Pull request merged**